docs/reference.rst

```diff
@@ -5,12 +5,22 @@ Reference
 
     Get QR code from a gray scale image ('Y800' mode)
 
+    The image should only have one QR code.
+
     :param bytes raw: Image data
     :param int width: Image width
     :param int height: Image height
     :return: The QR code value or None
 
 
+.. py:function:: scan_codes(symbology, image)
+
+    Get *symbology* codes from a PIL Image
```

Comment:
My dictionary says that *symbology* is "the study or the use of symbols and symbolism". Other disctionaries have additional definitions that may explain your use here, but I find the word obscure. Perhaps *code_type* would be a better name for the function argument (and in the description of the function as well)?

Also, we should add a list of allowed "code types". I would not know whether it's "QR" or "qr" (or perhaps both).